tests/test_utils.py

```diff
@@ -529,3 +531,13 @@ def test_add_dict_to_cookiejar(cookiejar):
     cj = add_dict_to_cookiejar(cookiejar, cookiedict)
     cookies = dict((cookie.name, cookie.value) for cookie in cj)
     assert cookiedict == cookies
+
+
+@pytest.mark.parametrize(
+    'value, expected', (
+                (u'test', True),
+                (u'ジェーピーニック', False),
+    )
+)
```

Comment:
I'd like to see a test here that uses something that's present in Latin1, as well, just to assuage concerns.